README.md

```diff
@@ -4,25 +4,84 @@
 
 To Contribute, please follow the below instructions.
 
-## Requirements
+### 1. Requirements
 
 - [Git](https://git-scm.com/) — latest source release
 - [Node.js](https://nodejs.org/) — latest LTS version or newer
+- [Hugo] (https://gohugo.io/) 
 
 
 ```bash
 git clone https://github.com/devopscloudjunction/website.git website && cd website
 ```
 
-
 ### 2. Install dependencies
 
 ```bash
 npm install
 ```
 
+Install Hugo
+
+```bash
+brew install hugo 
+```
+Check Hugo Version 
+
+```bash
+hugo version
+```
+
 ### 3. Start development server
 
 ```bash
 npm run start
-```
\ No newline at end of file
+```
+
+or you can use hugo for local server at http://localhost:1313 using below command
+
+```bash
+hugo server 
+```
+
+
+### Steps to make blog addition 
+
+1. Create a new branch out of 'main' branch using below git command and work in that branch
+
+```bash 
+git checkout -b sample-blog
+```
+2. Navigate to content/en/blog and create a directory for your blog. for example, 'test-blog'
+
+3. Copy sample index.md file from content/en/blog/say-hello-to-devopscloudjunction directory and sample image dcj.jpeg (images/dcj.jpeg) inside your created directory 'test-blog'
+
+or you can use below command to create a new blog dir and index.md
+
+```bash 
+hugo new blog/<blog-name>/index.yaml
```

Comment:
instead of <blog-name>, may be we can change to post name relevant to the article.

Reply:
yes the <blog-name> directory should be relevant like the title while the markdown file has to be index.md the code fails to render the image if anything else then index.md - tested a few times now.